tour_model_eval/viz_bins_clusters_above_cutoff.ipynb

```diff
@@ -0,0 +1,392 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "contemporary-bennett",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy\n",
+    "import emission.core.get_database as edb\n",
+    "import logging\n",
+    "import folium\n",
+    "import math\n",
+    "import branca.colormap as cm\n",
+    "import emission.storage.timeseries.abstract_timeseries as esta\n",
+    "import emission.analysis.modelling.tour_model.cluster_pipeline as eamtc\n",
+    "import emission.analysis.modelling.tour_model.similarity as similarity\n",
+    "import emission.analysis.modelling.tour_model.cluster_pipeline as pipeline\n",
+    "import emission.analysis.modelling.tour_model.featurization as featurization\n",
+    "import viz_bin_cluster as viz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "minus-norfolk",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger = logging.getLogger()\n",
+    "logger.setLevel(logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "configured-tobacco",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# copied from mode_purpose_share.ipynb\n",
+    "# We select participants from real data to be users at this time\n",
+    "participant_uuid_obj = list(edb.get_profile_db().find({\"install_group\": \"participant\"}, {\"user_id\": 1, \"_id\": 0}))\n",
+    "all_users = [u[\"user_id\"] for u in participant_uuid_obj]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "informative-mitchell",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logging.debug('There are %s users, we will plot the graphs for one of them' % len(all_users))\n",
+    "\n",
+    "user = all_users[1]"
```

Comment:
Is there a reason we are picking the second user?
Might want to add some comments around this choice.